lib/utils/note-utils.js

```diff
@@ -17,7 +17,7 @@ const noteTitleAndPreviewRegExp = /(\S[^\n]*)\s*(\S[^\n]*)?/;
  * whitespace and leads up to a newline or line end
  * @type {RegExp} matches a title and excerpt in note content skipping opening # from title
  */
-const noteTitleAndPreviewMdRegExp = /[# ]*(\S[^\n]*)\s*(\S[^\n]*)?/;
+const noteTitleAndPreviewMdRegExp = /(#+ +)?(\S[^\n]*)\s*(\S[^\n]*)?/;
```

Comment:
couple questions:
 - do we want to trap on `# # # # ` and friends?
 - valid headers are one through six long, should we trap that?
 - is a tab or quad-space different than the ASCII space?

```js
/(#{1,6}\s+)…
```

Reply:
good questions. 
- I think that `# # # #` should be displayed on the list as `# # #` because this is a proper behavior of markdown. 
- nice catch. I'll change that
- probably yes. \s+ is a better solution